RetrospectiveExtension.Frontend/package.json

```diff
@@ -65,6 +65,7 @@
     "@typescript-eslint/parser": "5.30.7",
     "@wojtekmaj/enzyme-adapter-react-17": "0.6.7",
     "autoprefixer": "10.4.7",
+    "cheerio": "^1.0.0-rc.3",
```

Comment:
I don't see we use "cheerio" in the project. is it added for future use?

Reply:
It's a related dependency from Jest, iirc from what I could gather based on some errors I was getting (https://github.com/microsoft/vsts-extension-retrospectives/actions/runs/3101588532/jobs/5023103595#step:5:301) - https://github.com/enzymejs/enzyme/issues/2518#issuecomment-1133447589